www/kelbi/src/styles/framework.css

```
@@ -29,3 +29,35 @@
 .text-center {
   text-align: center;
 }
+
+.w-full {
+  width: 100%;
+}
+
+.bg-gray-200 {
+  background-color: rgb(229 231 235);
```

```suggestion
  background-color: rgb(229, 231, 235);
```

same changes for others